processor/transformprocessor/internal/traces/functions.go

```diff
@@ -12,7 +12,10 @@ import (
 
 func SpanFunctions() map[string]ottl.Factory[ottlspan.TransformContext] {
 	// No trace-only functions yet.
-	return ottlfuncs.StandardFuncs[ottlspan.TransformContext]()
+	m := ottlfuncs.StandardFuncs[ottlspan.TransformContext]()
+	isRootSpanFactory := ottlfuncs.NewIsRootSpanFactory()
+	m[isRootSpanFactory.Name()] = isRootSpanFactory
+	return m
```

Comment:
We need to add this function to any contrib component that uses the ottlspan context and `internal/filter/filterottl`

Reply:
I missed that you added it to `filterottl` already, thanks!